Makefile

```diff
@@ -110,6 +110,7 @@ build-release:
 		-X $(ROOT)/pkg/version.gitTag=$(GIT_TAG)" \
 	    $(CMD_DIR)/$${target};                                                         \
 	done
+	@[[ ! -z "$(GIT_TAG)" ]] && echo "$(GIT_TAG)" > .GIT_TAG_INFO
```

Comment:
Will this file be packed into the source tar? I think it's better to implicitly add to the `MANIFEST.in`.

Reply:
added